db/webdataset/legacy_index_formats/README.md

```diff
@@ -0,0 +1,10 @@
+The data in this folder is used to test backwards compatibility
+with the WebDataset index file versions.
+
+Versions of wds2idx.py file used to generate particular .idx files.
+Calculated using md5sum (e.g., `$ md5sum wds2idx.py`).
+Should you need to generate .idx files for an older DALI version,
+please use these versions of the wds2idx.py script.
+
+v1.1 - 87b247f0a8d4e7af389278a2c8a360e1
```

Comment:
I'm not sure if MD5 is helpful for versioning. It does answer whether the version I have matches the expected one, but if I need to roll back to an older version, it's impossible to easily identify which version to select based on the given hash.